### ISSUE TITLE
Inconsistency in links api doc

### ISSUE THREAD

**Issue by uiureo** [ssbc/scuttlebot#356]
The doc of `links` api says `values (default: true)`, but the current behavior is `default: false`.

```
values (boolean, default: true): whether the data event should contain values. If set to true and keys set to false then data events will simply be values, rather than objects with a value property.
```

https://github.com/ssbc/scuttlebot/blob/e76ad26a3bc3baefb01277038df3744e64642cd3/api.md#links-source

Also, the api doc of  `secure-scuttlebutt` says `opts.values` is `default: false`.

https://github.com/ssbc/secure-scuttlebutt/blob/36140e6a6fde03200a46bbd30289135048ed8f71/README.md#securescuttlebuttlinks-source-feedid-dest-feedidmsgidblobid-rel-string-meta-true-keys-true-values-false-livefalse-reverse-false---pullsource

I could just create a pull request on this, but I'm not sure if you have some automated ways to update docs. So I just report it for now.

**Comment by dominictarr** [comment]
good spotting, fixed in cb2b30a2e3de13ab942b67edb29d63cda3292acd

**Comment by uiureo** [comment]
Thanks!
